### PR TITLE
Add whisper feature extractor

### PR DESCRIPTION
**Pull request by yuekaizhang**

### CODE REVIEW

**Comment by yuekaizhang** [comment]
@pzelasko Hi Piotr, I add a whisper feature extractor to better support whisper fine-tuning with lhotse. Would you mind checking it please? Thanks.

File: lhotse/features/whisper_fbank.py [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+
+@register_extractor
+class WhisperFbank(FeatureExtractor):
+    name = "whisper-fbank"
+    config_type = WhisperFbankConfig
+
+    def __init__(self, config: Optional[WhisperFbankConfig] = None):
+        super().__init__(config=config)
+        config_dict = self.config.to_dict()
+        config_dict.pop("device")
```
Comment (csukuangfj) [inline lhotse/features/whisper_fbank.py:112]: These two lines do not have any side effects. Can we remove them?

File: lhotse/features/whisper_fbank.py [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+
+    return log_spec
+
+
+@dataclass
+class WhisperFbankConfig:
+    sampling_rate: int = 16000
+    num_filters: int = 80
+    hop_length: int = 160
+    n_fft: int = 400
```
Comment (csukuangfj) [inline lhotse/features/whisper_fbank.py:93]: Whisper only supports these values and no other values can be used. Shall we hard-code them and don't make them configurable?

**Review by pzelasko — COMMENTED** [review]
Thanks, looks good! Can you add a unit test to cover this extractor? You can look either at `test/features/*.py` or `test/cut/test_feature_extraction.py` for starting points.

**Comment by yuekaizhang** [comment]
> Thanks, looks good! Can you add a unit test to cover this extractor? You can look either at `test/features/*.py` or `test/cut/test_feature_extraction.py` for starting points.

Done.

**Review by pzelasko — APPROVED** [review]
LGTM

**Pull request merged**